projects/plugins/boost/app/features/optimizations/render-blocking-js/class-render-blocking-js.php

```diff
@@ -210,7 +210,7 @@ protected function ignore_exclusion_scripts( $buffer ) {
 			'~<!--.*?-->~si',
 
 			// Scripts with application/json type
-			'~<script.*type=(?<q>["\']*)application/json\k<q>.*>.*</script>~si',
+			'~<script\s+[^\>]*type=(?<q>["\']*)application/json\k<q>.*>.*</script>~si',
```

Comment:
The following will still fail:
```
<script>alert('hello');</script>
<script type="application/json">{}</script>
Hello World!
<script>alert('hello');</script>
```

**Suggestion**
```suggestion
			'~<script.*?type=(?<q>["\']*)application/json\k<q>.*?>.*?</script>~si',
```

Reply:
Thank you. Fixed in 9e9757cba2 - though I didn't take your exact suggestion.

```
<script>alert('hello');</script>
<script type="application/json">{}</script>
Hello World!
<script>alert('hello');</script>
```

Still should match the middle one. I went with this:
```
'~<script\s+[^\>]*type=(?<q>["\']*)application/json\k<q>.*?>.*?</script>~si'
```